test/e2e-helm/e2e_helm_project_locally_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e_helm_test
+
+import (
+	"os/exec"
+
+	. "github.com/onsi/ginkgo" //nolint:golint
+	. "github.com/onsi/gomega" //nolint:golint
+)
+
+var _ = Describe("Running Helm projects", func() {
+	Context("built with operator-sdk", func() {
+
+		AfterEach(func() {
+			By("Uninstalling CRD's")
+			_ = tc.Make("uninstall")
+		})
+
+		It("Should generate a runnable project locally", func() {
+			By("Installing CRD's")
+			err := tc.Make("install")
+			Expect(err).NotTo(HaveOccurred())
+
+			cmd := exec.Command("make", "run")
+			err = cmd.Start()
+			Expect(err).NotTo(HaveOccurred())
+
```

Comment:
We _probably_ want to add similar scenarios here that we have in the cluster test. But that can be in a follow-up.